.github/workflows/sims.yml

```diff
@@ -79,48 +79,3 @@ jobs:
       - name: test-sim-multi-seed-short
         run: |
           make test-sim-multi-seed-short
-
-  sims-notify-success:
```

Comment:
It seems ideal to continue to run the sim tests above, but these two jobs require a `SLACK_WEBHOOK` which pipe failures and successes into slack. I think this would just be noise for us so I'd prefer not to do this.